acid/features/zuul_manager/tests/test_control_panel.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+from unittest import mock
+
+import requests
+
+from acid.config import config
+from acid.features.auth.tests.fixtures import UserFactory
+from acid.tests import IntegrationTestCase
+
+
+@mock.patch('acid.features.auth.service.fetch_user_data')
```

Comment:
Please add decorators for unit & integration tests, like in https://github.com/codilime/acid/pull/10